ui/apps/akasha/src/components/__tests__/feed-page.spec.tsx

```diff
@@ -51,4 +53,32 @@ describe('< FeedPage /> component', () => {
     expect(screen.getAllByTestId('avatar-image')).not.toBeNull();
     expect(screen.getByText(/Share your thoughts/i)).toBeInTheDocument();
   });
+
+  it('should show draft post from local storage and clear it', async () => {
+    const loginState = genLoggedInState(true);
+    localStorage.setItem(
```

Comment:
I think we should mock localStorage(maybe with Map object), we don't need to test if the localStorage is saving the data properly. It also saves us from having to do cleanup after running the tests(clear item from localStorage)

Reply:
Hmm actually I am not testing if the localStorage is saving the data properly. This are two things, one is I am testing the feed page showing the saved draft post. And then testing clearing of this saved draft post.

To avoid this confusion, I've split the test case into two.

Reply:
I've mocked the localStorage based on what we discussed on the review call.